### PR TITLE
use json5 and add inline comments to config

### PR DESCRIPTION
**Pull request by emortong**
[Ticket CPP-561](https://financialtimes.atlassian.net/browse/CPP-561)

Adding inline comments and using JSON5 as part of the actions that came up from the [Audit & document Renovate config ticket](https://financialtimes.atlassian.net/browse/CPP-556)

[Notion Renovate audit notes](https://www.notion.so/apaleslimghost/Renovate-config-audit-53466f032b824d5cae4a2a71324c86b2)

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      ],
+      dependencyDashboardApproval: true,
+    },
+    {
+      // group n-gage & n-heroku tools into one PR and opens a PR without approval
+      packageNames: [
+        '@financial-times/n-gage',
+        '@financial-times/n-heroku-tools',
+      ],
+      dependencyDashboardApproval: false,
```
Comment (apaleslimghost) [inline renovate.json5:66]: **suggestion**: i think this should mention that this is skipping the approval set by the previous rule

File: renovate.json5 [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        'fetchres',
+        'isomorphic-fetch',
+      ],
+      depTypeList: [
+        'dependencies',
+      ],
+      rangeStrategy: 'auto',
+    },
+    {
+      // for devDependencies, don't create a PR until the checkbox is checked on the "master issue"
```
Comment (apaleslimghost) [inline renovate.json5:54]: **nitpick**: we should call this "dependency dashboard"

```suggestion
      // for devDependencies, don't create a PR until the checkbox is checked on the "dependency dashboard" issue
```

**Review by apaleslimghost — APPROVED** [review]
😍 fantastic! couple of non-blocking comments

File: renovate.json5 (hunk 2 above)
Comment (emortong) [reply renovate.json5:54]: good catch, thanks

**Pull request merged**